README.md

```diff
@@ -281,6 +287,30 @@ The Client ID and the Client Secret will be needed for the basic script you will
    
 You can now restart your Flexy to restart the Java application. Messages will be sent to your OCS endpoint.
 
+## JVM Logger
+The OSIsoft connector makes use of the [HMS Solution Center logging library](https://github.com/hms-networks/sc-flexy-logger-lib) to ensure all logging follows HMS standards for general connectors.
```

Comment:
> all logging follows HMS standards for general connectors

What does that mean?

Reply:
just saying it uses the library sounded to short haha.
will change to:
The OSIsoft connector makes use of the [HMS Solution Center logging library](https://github.com/hms-networks/sc-flexy-logger-lib)